### PR TITLE
EWPP-4431: Update OP vocabularies.

### PR DESCRIPTION
**Pull request by sergepavle**

### CODE REVIEW

**Review by brummbar — DISMISSED** [review]
Blocking until new releases is done after #109 .

**Review by intelektron — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,16 +11,16 @@ based on [OpenLink Virtuoso](https://virtuoso.openlinksw.com).
 The following RDF triples will be imported once the service starts:
 
 - [Corporate body classification](https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/corporate-body-classification&version=20230927-0)
-- [Corporate body](https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/corporate-body&version=20240321-0)
+- [Corporate body](https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/corporate-body&version=20240612-0)
 - [Country](https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/country&version=20230614-0)
```
Comment (imanoleguskiza) [inline README.md:15]: The country vocabulary is not up to date: https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/country&version=20240612-0

File: README.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ The following RDF triples will be imported once the service starts:
 - [EU political leader name](https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/political-leader&version=20240111-0)
```
Comment (imanoleguskiza) [inline README.md:31]: Not up to date: https://op.europa.eu/en/web/eu-vocabularies/dataset/-/resource?uri=http://publications.europa.eu/resource/dataset/political-leader&version=20240612-0

**Review by imanoleguskiza — COMMENTED** [review]
Two vocabularies are not in their latest version

**Review by imanoleguskiza — APPROVED** [review]

**Pull request merged**